### DISCUSSION TITLE
Trick for keeping phone awake for large initial sync to complete?

### DISCUSSION THREAD

**Discussion by mlwelles** [MobiusSync/MobiusSync#62] [Ideas]
Even in foreground, I’m having issues getting iOS to complete the initial sync of a large folder.  Unless I’m interacting with the screen it’ll eventually turn off the display and the app will be suspended.

Have you thought about adding anything to enable tricking iOS into keeping the app open?

Like if you added a button that’d start full screen playback of [this 11 hour video on the youtube](https://youtu.be/G52dUQLxPzg) 

The phone should stay awake for the entire playback without needing any interaction.  Instead  of having to keep tapping on the screen,  I could just watch pretty fish swim by.  

Or, even better, go do something else  knowing the sync would be able to complete without me. 

It could even be a branding opportunity - you could get fancy and have it playback an 11 hour video of animated moebius strip :)

**Comment by 497e0bdf29873** [comment]
Go to iOS Settings -> Display & Brightness, and set Auto-Lock: never. For security reasons, restore auto-locking after sync is complete. Rarely the app may also crash.

**Comment by SimonPickup** [comment]
This is enhancement request #31
